packages/build/src/plugins_core/edge_functions/validate_manifest/validate_edge_functions_manifest.js

```diff
@@ -42,6 +42,18 @@ const routesSchema = {
   additionalProperties: false,
 }
 
+const layersSchema = {
+  $async: true,
+  type: 'object',
+  required: ['flag', 'name'],
+  properties: {
+    flag: { type: 'string' },
+    name: { type: 'string' },
+    local: { type: 'string' },
```

Comment:
I do not see this field in edge-bundler?

Reply:
It'll be added in a follow-up PR, as explained in https://github.com/netlify/edge-bundler/pull/198.

I figured I'd make it available in Build now to avoid having to make two PRs.